site.yml

```
@@ -21,460 +21,460 @@ Documents:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/README.md
         Slug: dotnet-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client Quickstart Guide
+        SEO Description: The MinIO .NET Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: dotnet, .NET
 
       - .NET Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/Docs/API.md
         Slug: dotnet-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client API Reference
+        SEO Description: The MinIO .NET Client SDK provides detailed code examples for the .NET
+        SEO Keywords: dotnet, .NET
 
   - Cookbook:
       - S3cmd with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/s3cmd-with-minio.md
         Slug: s3cmd-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | S3cmd with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to configure and use S3cmd to manage data with MinIO Server
+        SEO Keywords: S3cmd
 
       - AWS CLI with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-cli-with-minio.md
         Slug: aws-cli-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | AWS CLI with MinIO- Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to configure and use AWS CLI to manage data with MinIO Server
+        SEO Keywords: AWS CLI
 
       - restic with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/restic-with-minio.md
         Slug: restic-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | restic with MinIO- Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use restic to backup data into MinIO Server.
+        SEO Keywords: Restic
 
       - Store MySQL Backups in Minio:
         Link: https://github.com/minio/cookbook/blob/master/docs/store-mysql-backups-in-minio.md
         Slug: store-mysql-backups-in-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store MySQL Backups in MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to store MySQL backups in MinIO Server.
+        SEO Keywords: MySQL, Backups
 
       - Store MongoDB Backups in Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/store-mongodb-backups-in-minio.md
         Slug: store-mongodb-backups-in-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store MongoDB Backups in MinIO- Cookbook/Recipe
```

Can we have a space between MinIO and the slash as follows?

```suggestion
        SEO Title: MinIO | Store MongoDB Backups in MinIO - Cookbook/Recipe
```